dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -605,13 +606,19 @@ export const Card = ({
 						imagePositionOnMobile={imagePositionOnMobile}
 						showPlayIcon={showPlayIcon}
 					>
-						{media.type === 'slideshow' && (
-							<Slideshow
-								images={media.slideshowImages}
-								imageSize={imageSize}
-								isDynamo={isDynamo}
-							/>
-						)}
+						{media.type === 'slideshow' &&
+							(isFlexibleContainer ? (
+								<SlideshowCarousel
+									images={media.slideshowImages}
+									imageSize={imageSize}
+								/>
+							) : (
```

Comment:
@cemms1 I accidentally left a 'pending' comment here so you wouldn't have seen it when you reviewed this! Essentially it was questioning whether I should remove this for now given that the Card component overlays prevent the carousel from being scrolled so if viewed in situ on one of the test fronts it appears as a static image. As the component is under development I might remove this for now, and reinstate when it's more functional.